www/source/partials/docs/_using-builder-on-prem.html.md.erb

```diff
@@ -1,5 +1,7 @@
 ## <a name="on-prem" id="on-prem" data-magellan-target="on-prem">Install and Use Builder On-Premises</a>
 
-In addition to our hosted service, we also support installing and running Habitat Builder on-premises, using your own network and infrastructure, which allows you to choose from a wider selection of authentication providers and to manage how Builder fits into your existing CI/CD processes.
+In addition to our hosted service, we also support installing and running a Habitat Builder Depot on-premises, using your own network and infrastructure, which allows you to choose from a wider selection of authentication providers and to manage how Builder fits into your existing CI/CD processes.
+
+Currently, our on-premises Builder depot only stores packages for download and upload by Supervisors and Studios. We do intend to bring the full capabilities of Habitat Builder to our on premises option in the future.
```

Comment:
Initially I read this as "We don't intend to bring" and had to re-read it. Consider removing the "do" in "We do intend"